azafea/event_processors/endless/metrics/events/_base.py

```diff
@@ -64,6 +64,7 @@
 IGNORED_EMPTY_PAYLOAD_ERRORS: Set[str] = {
     '9af2cc74-d6dd-423f-ac44-600a6eee2d96',
     'add052be-7b2a-4959-81a5-a7f45062ee98',
+    'eb0302d8-62e7-274b-365f-cd4e59103983',
```

Comment:
```suggestion
    # LocationLabel
    'eb0302d8-62e7-274b-365f-cd4e59103983',
```

Reply:
(I know the others don't have comments.)

Or could this fetch the UUID property from that class? I think not, because that would be a circular import.

Reply:
> Or could this fetch the UUID property from that class? I think not, because that would be a circular import.

The circular import would be hard to avoid, I think that’s too much work for what we want to get.

But we can:
- Add a comment, even at the end of each line (event IDs have the same lengths, that would be readable).
- Fill `IGNORED_EVENTS` and `IGNORED_EMPTY_PAYLOAD_ERRORS` with the metaclass, like for the `*_EVENT_MODELS` dicts above.

I like the metaclass solution because it would put the information where it belongs: in the table classes. I could do that in another PR.

Reply:
Yep, I agree that that could be a separate PR.

Reply:
#117 can be merged after #115.